src/components/Compute/compute-status-controller/compute-query-controller.ts

```diff
@@ -0,0 +1,42 @@
+const Long = require("long");
```

Comment:
can we change it to import? try this
```suggestion
import * as Long from "long";
```